README.md

```diff
@@ -66,3 +76,16 @@ Plugin 'christoomey/vim-system-copy'
 " ...
 call vundle#end()
 ```
+
+An alternative would be to use [Plug](https://github.com/junegunn/vim-plug).
+As done with Vundle, simply add to your `.vimrc`:
+
+``` vim
+call plug#begin()
+" ...
+Plug 'christoomey/vim-system-copy'
+" ...
+call plug#end()
```

Comment:
I'm a plug user myself these days, so I'd be fine with removing the Vundle section and just replacing with thus plug reference. Mind making that change?